File: lesson_1/linearSearch.js
/** Алгоритм линейного поиска */

function linearSearch() {
    let arr = [1, 2, 3, 4, 5, 6];
    console.log(`Number exists(js)? = ${getNumber(arr, 5)}`);
}

function getNumber(arr, number) {
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] === number) {
            return true;
        }
    }
    return false;
}

//console.log("Test");

linearSearch();

File: lesson_1/LinearSearch.java
public class LinearSearch {
    public static void main(String[] args) {
        int[] arr = {1, 2, 3, 4, 5, 6};
        System.out.println("Number exists(java)? - " + getNumber(arr, 4));
    }

    private static boolean getNumber(int[] arr, int number) {
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == number) {
                return true;
            }
        }
        return false;
    }
}
